spec/rubocop/cop/rspec/file_path_spec.rb

```diff
@@ -116,8 +116,17 @@
   it 'handles alphanumeric class names' do
     inspect_source(
       cop,
-      'describe IPv4AndIPv6 do; end',
-      'i_pv4_and_i_pv6_spec.rb'
+      'describe IPV4AndIPV6 do; end',
+      'ipv4_and_ipv6_spec.rb'
+    )
+    expect(cop.offenses).to be_empty
+  end
+
+  it 'handles more alphanumeric class names' do
```

Comment:
This 2nd spec was nice to have when developing the feature, but it can probably be removed now. What do you think?